gui-apps/river-tag-overlay/river-tag-overlay-1.0.0.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Gentoo Authors
```

Comment:
Please add your preferred name to this. Take a look at some other ebuilds in this repo.
They are using the ISC license.

Reply:
Sorry about that one. I got very excited when the ebuild finally worked and totally forgot to proofread and remove obvious oversights before submitting :/